lib/_io.scm

```diff
@@ -8343,6 +8343,167 @@
         (macro-dynamic-bind output-port port thunk)))))
 
 ;;;----------------------------------------------------------------------------
+(define-prim (##open-raw-device
+              direction
+              name
+              fd)
```

Comment:
Will rename to `device` and call it `##make-raw-device`. Will also do the empty line.